docs/source/raft_configuration.md

```diff
@@ -86,22 +86,28 @@ other, and TLS certificates issued by a public TLS CA for the client facing API.
 
   * `ClientCertificate`, `ClientPrivateKey`: The file path of the client TLS certificate
   and corresponding private key.
-  * `ListenPort`: The port the cluster listens on. If blank, the port is the same
-  port as the orderer general port (`general.listenPort`)
+  * `ListenPort`: The port the cluster listens on. 
+  It must be same as `consenters[i].Port` in Channel configuration. 
```

Comment:
Not sure if this is the best place to mention this. It feels like this belongs in the section describing the Consenters but that would require reordering some sections of this doc. The natural flow to me would be to describe the configuration of a single raft node and then how it fits into the channel configuration, not vice versa.

Reply:
Agree, I see this difficulty in how to express how they are bound. If another maintainer also agree on we are better to reorganize sections then I am glad to change accordingly.